### PR TITLE
[DOCS] Update runbook-vet-center-name-change.md

### PR DESCRIPTION
**Pull request by davidconlon**
There are two cases that we put in here as well as some questions that we had about process, so requesting Swirt review to contribute

## Description

Relates to #_issueid_. (or closes?)

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As user _uid_ with _user_role_
1. Do this
   - [ ] Validate that
2. Then
   - [ ] Validate that
3. Then validate Acceptance Criteria from issue
   - [ ] a
   - [ ] b
   - [ ] c

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md [hunk 1]
```diff
@@ -47,11 +47,25 @@ _Help desk will complete these steps or escalate to request help from CMS engine
...
+- [ ] 5. The new official name no longer matches the pattern "<location> Vet Center" so you need to change the common name to align with the <location> naming convention. (Generally this will already be the previous name and you may not need to change anything? plz verify)
+- [ ] Confirm on the Front-end that the <location> naming convention is the h1 followed by a "also known as" with the named-after-person name.
+
+**Proceed to step 10**
+
+***CASE: Renamed, but still location-based naming convention***
+
 - [ ] 5. In the [Sections taxonomy](https://prod.cms.va.gov/admin/structure/taxonomy/manage/administration/overview), Vet Centers: rename the Section for this Vet Center to match its new name
-- [ ] 6. If the new official name matches the pattern "<city> Vet Center", update the common name to match
+- [ ] 6. If the new official name matches the pattern "<city> Vet Center", update the common name to match **Does this have to happen? I thought we could leave this blank? Plz confirm**
```
Comment (swirtSJW) [inline .github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md:61]: The common name is a required field and must be completed in order for a human to save the node 
image


The FE bases its logic to show the "aka" if these two fields have different values.

Comment (swirtSJW) [inline .github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md:61]:
```suggestion
- [ ] 6. If the new official name matches the pattern "<city> Vet Center", update the common name to match.
```

File: .github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md [hunk 2]
```diff
@@ -47,11 +47,25 @@ _Help desk will complete these steps or escalate to request help from CMS engine
 - [ ] 4. Verify which Vet Center it belongs to, and confirm that the "Main Vet Center Location" field is set correctly.
 
 **If a Vet Center**
+
+***CASE: Renamed, but named after a person, not a location***
+
+- [ ] 5. The new official name no longer matches the pattern "<location> Vet Center" so you need to change the common name to align with the <location> naming convention. (Generally this will already be the previous name and you may not need to change anything? plz verify)
```
Comment (swirtSJW) [inline .github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md:53]:
```suggestion
- [ ] 5. The new official name no longer matches the pattern "<location> Vet Center" so you need to change the common name to align with the <location> naming convention if it does not already.
```

File: .github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md [hunk 3]
```diff
@@ -47,11 +47,25 @@ _Help desk will complete these steps or escalate to request help from CMS engine
...
 - [ ] 5. In the [Sections taxonomy](https://prod.cms.va.gov/admin/structure/taxonomy/manage/administration/overview), Vet Centers: rename the Section for this Vet Center to match its new name
-- [ ] 6. If the new official name matches the pattern "<city> Vet Center", update the common name to match
+- [ ] 6. If the new official name matches the pattern "<city> Vet Center", update the common name to match **Does this have to happen? I thought we could leave this blank? Plz confirm**
 - [ ] 7. Visit [bulk operations](https://prod.cms.va.gov/admin/content/bulk) page and filter by section = vet center name
 - [ ] 8. Update URLs for all content in that section by bulk operations
 - [ ] 9. Resave all content in that section by bulk operations
+
+**Do we need to do some sort of bulk update to any CAPs names?**
+
+
```
Comment (swirtSJW) [inline .github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md:68]:
```suggestion
```

I tested it out, CAPs get renamed and re-aliased as part of steps 7-9 above.  No extra effort required.

**Review by omahane — DISMISSED** [review]
This looks good with the suggestions by @swirtSJW

File: .github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md (hunk 1 above)
Comment (omahane) [reply .github/ISSUE_TEMPLATE/runbook-vet-center-name-change.md:61]: It's not added as a suggestion, but I recommend removing "**Does this have to happen? I thought we could leave this blank? Plz confirm**"

**Review by swirtSJW — APPROVED** [review]

**Pull request merged**